File: 125-valid-palindrome/valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        s1=s.lower()
        em=""
        for i in s1:
            if i.isalnum():
                em+=i
        rev=em[::-1]
        if rev==em:
            return True
        else:
            return False

File: 125-valid-palindrome/valid-palindrome.java
class Solution {
    public boolean isPalindrome(String s) {
        StringBuilder sb1 = new StringBuilder();
        

        for(char i:s.toCharArray()){
            if(Character.isLetterOrDigit(i)){
                sb1.append(Character.toLowerCase(i));
            }
        }

        int i=0;
        int j=sb1.length()-1;
        while(i<=j){
            if(sb1.charAt(i)==sb1.charAt(j)){
                i++;
                j--;
            }
            else{
                return false;
            }
            
        }
        return true;
    }
}